server/server.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+const port = process.env.PORT || 5000;
+
+const events = [
+
+];
+
+let idCounter = 0;
+
+app.use(bodyParser.json());
+
+app.post('/events', function test2(req,res) {
+  idCounter += 1;
+  const eventObject = {
+    id: idCounter,
+    name: req.body.name
+  };
+  events.push(eventObject);
+  console.log(events);
+  res.status(201).send({ eventObject });
+});
+
+app.delete('/events/:id',(req, res) => {
+  const requestId = events.find((event) => event.id === req.params.id);
```

Comment:
You're finding the event, but it doesn't look like you're actually deleting the event.